fs-sim/src/System/FS/Sim/Error.hs

```diff
@@ -40,26 +40,32 @@ module Test.Util.FS.Sim.Error (
 
 import           Prelude hiding (null)
 
+import           Control.Concurrent.Class.MonadSTM.Strict
 import           Control.Monad (replicateM, void)
+import           Control.Monad.Class.MonadThrow hiding (handle)
 
+import           Data.ByteString (ByteString)
 import qualified Data.ByteString as BS
+import qualified Data.ByteString.Char8 as C8
+import qualified Data.ByteString.Lazy as BL
+import qualified Data.ByteString.Lazy.Char8 as LC8
+import qualified Data.List as List
 import           Data.List (dropWhileEnd, intercalate)
 import           Data.Maybe (catMaybes, isNothing)
+import           Data.String (IsString (..))
 import           Data.Word (Word64)
 
 import qualified Test.QuickCheck as QC
-import           Test.QuickCheck (Arbitrary (..), Gen)
+import           Test.QuickCheck (ASCIIString (..), Arbitrary (..), Gen,
+                     suchThat)
 
-import           Ouroboros.Consensus.Util (whenJust)
-import           Ouroboros.Consensus.Util.CallStack
-import           Ouroboros.Consensus.Util.IOLike hiding (handle)
+import           Util.CallStack
 
-import           Ouroboros.Consensus.Storage.FS.API
-import           Ouroboros.Consensus.Storage.FS.API.Types
+import           System.FS.API
```

Comment:
Would it make sense to re-export symbols in `.Types` so that we only use one `import`?

Reply:
I was thinking that a `System.FS` module that exports all the relevant stuff would probably be useful, so that for most cases just importing that one module is sufficient. It will probably in a follow-up PR